app/models/SearchHits.ts

```diff
@@ -35,17 +35,24 @@ export interface OrganizationHit
 }
 export type SearchResultsResponse = AlogliaSearchResultsType<SearchHit>;
 export type SearchHit = ServiceHit | OrganizationHit;
+type Location = {
```

Comment:
My thought here is to distinguish `Address` data from the concept of a  _location_ on a map. We don't need to pass through all the address data, we want an intermediate representation.